lib/core-client/src/preview/start.ts

```diff
@@ -76,6 +76,9 @@ export function start<TFramework extends AnyFramework>(
     // eslint-disable-next-line no-underscore-dangle
     globalWindow.__STORYBOOK_PREVIEW__ = preview;
     globalWindow.__STORYBOOK_STORY_STORE__ = preview.storyStore;
+
+    // To enable user code to detect if it is running in Storybook
+    globalWindow.IS_STORYBOOK = true;
```

Comment:
This only gets executed in legacy 6.x mode, and not if you're using `features.storyStoreV7`

Reply:
@shilman I see that now, thank you. Is it okay to just move my line of code above the `if (FEATURES?.storyStoreV7)`? 

Yann directed me to this file but I am a bit confused because `start.ts` does basically nothing if StoryStoreV7 is enabled.

Reply:
sorry missed this question first time around. I think moving this to the top of the file will work. you should also check `if (globalWindow) ...` (i'm not sure when it isn't set, but i'm sure the code is there for a reason)